packages/core/src/Avatar.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
```

Comment:
I think new core components should be added to v2 as well, so we can have backwards compatibility in v3.

Reply:
You could look at it as incentive to move to v3 😜

Reply:
I'm looking forward to inheriting the performance wins in v3 for sure. I still strongly recommend putting these components in v2. When working in a host app that consumes several dependencies that all need to switch over from v2 to v3, backwards compatibility really helps facilitate that transition.

Reply:
Yeah that totally makes sense, thanks for bringing this up. Here is the PR for v2 Avatar https://github.com/priceline/design-system/pull/678 👍

Reply:
PR for Breadcrumbs v2 as well https://github.com/priceline/design-system/pull/677!